Makefile

```diff
@@ -241,4 +261,7 @@ prepare-test-env:
 	echo "Step 4: Preparing the storage service..."; \
 	$(MAKE) prepare.storage; \
 
+	echo "Step 5: Preparing the k6 load node..."; \
+	$(MAKE) up/testing; \
```

Comment:
Should we wait for some time or check something after starting a k6 container?

Reply:
There is nothing to wait there, but I added 5 secs just in case